it.js

```diff
@@ -358,7 +358,7 @@ settings.viewmodel.weapSwapY Scambia posizione Y arma
 settings.viewmodel.weapReloadY Posizione Y in ricarica
 settings.viewmodel.yOffADS Posizione Y arma quando miri
 settings.viewmodel.adsFovMlt Intensità FOV quando miri
-settings.viewmodel.toggleADS Toggle Weapon ADS
+settings.viewmodel.toggleADS Attiva/Disattiva arma in ADS
```

Comment:
This refers to the ability of activating the ADS itself so it would be best if translated as "Attiva ADS" or "Attiva mira"/"Attiva zoom mira".